src/Middleware/CORS/test/FunctionalTests/CORS.FunctionalTests.npmproj

```diff
@@ -12,10 +12,5 @@
     <IsTestProject>false</IsTestProject>
   </PropertyGroup>
 
-  <Target Name="InstallPuppeteer" BeforeTargets="Restore">
-    <!-- Explicitly install puppeteer. This will install the bundled Chromium as part of restore instead of as part of execution -->
-    <Yarn Command="add puppeteer" />
```

Comment:
> The puppeteer package is already part of the package.json file for the CORS functional tests.

Here's the reason it was previously added: https://github.com/GoogleChrome/puppeteer/issues/1602#issuecomment-355545915. I'm happy to consider this if it works without it.

Reply:
Tests pass on CI and I don't see the "Chromium revision is not downloaded" error. Is that sufficient validation?

By the way, we are already installing Chrome globally for some Kestrel tests. Could we switch the lighter version of this package, puppeteer-core, which doesn't bundle Chrome? This saves ~170MB Mac, ~282MB Linux, ~280MB Win.

Reply:
Their documentation says it's not guaranteed to work with anything outside the bundled version, but if it works, I'm happy with that.

Reply:
Let's give it a try. If things start failing, I'll revert this piece.